packages/nyc-config-typescript/index.json

```diff
@@ -0,0 +1,17 @@
+{
+  "cache": false,
```

Comment:
I'm don't know why you would want to disable nyc cache when using ts-node but I also don't know enough to object to it.

Reply:
I've seen some issues pop up as a result of `cache: true`, so, just for the tutorial/starting package it's probably better. It can of course be overridden in `.nycrc` etc